migrations/migrate.sh

```diff
@@ -7,45 +7,49 @@ iter=0
 
 case $SERVICE_NAME in
 	notary_server)
+    MIGRATIONS_PATH=${MIGRATIONS_PATH:-migrations/server/mysql}
+    DB_URL=${DB_URL:-mysql://server@tcp(mysql:3306)/notaryserver}
 		# have to poll for DB to come up
-		until migrate -path=migrations/server/mysql -url="mysql://server@tcp(mysql:3306)/notaryserver" version > /dev/null
+		until migrate -path=$MIGRATIONS_PATH -url=$DB_URL version > /dev/null
 		do
 			iter=$(( iter+1 ))
 			if [[ $iter -gt 30 ]]; then
 				echo "notaryserver database failed to come up within 30 seconds"
 				exit 1;
 			fi
-			echo "waiting for notarymysql to come up."
+      echo "waiting for $DB_URL to come up."
```

Comment:
Looks like you're using spaces here (and a couple of other places), should be tabs in shell scripts :smile: